pkg/test/flags.go

```diff
@@ -72,6 +73,15 @@ func processFlags() error {
 		os.Exit(0)
 	}
 
+	// TODO: Instead of using hub/tag, we should be using the local registry to load images from.
+	// See https://github.com/istio/istio/issues/6178 for details.
+
+	// Capture environment variables
+	hub := os.Getenv("HUB")
```

Comment:
Hub and tag are cluster-specific settings.  I don't think they belong in a common data structure.  Sooner or later we're going to have to hash out how to configure environment-specific settings ... might be worth thinking about now.

In addition, for cluster-based tests we may want a different hub/tag per component (e.g. mixer, pilot, ca, app). Perhaps we should have some way of configuring individual components, which are environment-specific?

Reply:
Discussed offline: Algo is:
- Use local images by default
- Allow explicit image specification.
- If not specificed, fall back to local images.